.travis.yml

```diff
@@ -0,0 +1,28 @@
+sudo: required
+
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - gcc-4.8
+      - g++-4.8
+      - wget
+      # Package list from http://bazel.io/docs/install.html
+      - oracle-java8-installer
+      - pkg-config
+      - zip
+      - zlib1g-dev
+
+install:
+  - export JAVA_HOME=/usr/lib/jvm/java-8-oracle
+  - export CC=gcc-4.8
+  - export CXX=g++-4.8
+  - wget 'https://github.com/bazelbuild/bazel/releases/download/0.4.1/bazel-0.4.1-installer-linux-x86_64.sh' -O ./bazel-installer.sh
+  - chmod +x ./bazel-installer.sh
+  - ./bazel-installer.sh  --user
+
+script:
+  - script/setup
```

Comment:
Still testing it, as I couldn't enable travis for this repo, asked @mandarjog for help. Can you enable it?